api/controllers/CommunityController.js

```diff
@@ -1,3 +1,9 @@
+var Promise = require('bluebird'),
+  request = require('request'),
```

Comment:
please make sure that this matches Javascript Standard Style (no semicolons, each var declaration on its own line, etc)